src/ai/susi/mind/SusiSkill.java

```diff
@@ -525,12 +525,42 @@ public static JSONObject getSkillMetadata(String model, String group, String lan
                     JSONObject languageName = groupName.getJSONObject(language);
                     if (languageName.has(skillname)) {
                         JSONObject skillName = languageName.getJSONObject(skillname);
+                        if (!skillName.has("stars")){
+                            JSONObject newFiveStarRating=new JSONObject();
+                            newFiveStarRating.put("one_star", "0");
+                            newFiveStarRating.put("two_star", "0");
+                            newFiveStarRating.put("three_star", "0");
+                            newFiveStarRating.put("four_star", "0");
+                            newFiveStarRating.put("five_star", "0");
+                            newFiveStarRating.put("avg_star", "0");
+                            newFiveStarRating.put("total_star", "0");
+
+                            skillName.put("stars", newFiveStarRating);
```

Comment:
While this is nice, I am not sure if having `string` as the value of the map is a good idea. Can this instead be `int/Integer` values accross the `one_star`...... etc.., This also makes it easy to do mathematical operations on the items in these stars.